src/plugin/lib-selenium/src/java/org/apache/nutch/protocol/selenium/HttpWebClient.java

```diff
@@ -166,7 +175,7 @@ public static String getHTMLContent(WebDriver driver, Configuration conf) {
   public static void cleanUpDriver(WebDriver driver) {
     if (driver != null) {
       try {
-	      driver.close();
+//	      driver.close();
```

Comment:
@hussein-alahmad Why is this commented?

Reply:
there is a driver.quit() statement after it , I was getting an error about trying to close a non existing connection , after searching about it , turned out that driver.quit() alone without the driver.close() do the work 
as mentioned [here](https://github.com/mozilla/geckodriver/issues/966)

Reply:
Thank you @hussein-alahmad